_posts/2015-02-18-ApiBlueprint-button.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Social blueprint
+excerpt: Share your blueprint with others and learn from masters.
+layout: post
+date: 2015-02-18 00:00:00 +0100
+author: vincenzo
+published: true
+commets: true
+---
+
+Recently, we have received a lot of feedback from users that would like to learn more about API Blueprint and how to present stunning and beautiful documentation for their users.
+
+We care about your feedback and we think that documentation and tutorials about [API Blueprint](http://apiblueprint.org) are extremely important. Given that we're hard working on this, and that tons of stuff is coming, today I am glad to announce that we've made another step forward in that direction.
```

Comment:
You may just remove the entire sentence:

> Given that we're hard working on this, and that tons of stuff is coming, today I am glad to announce that we've made another step forward in that direction.

You are basically saying this same thing in the next sentence, just with more clarity.